internal/client.go

```diff
@@ -38,7 +38,7 @@ import (
 )
 
 const (
-	clientVersion        = "v2.1.0"
+	clientVersion        = "v2.1.1"
```

Comment:
the 2.1.1 has been releases, it seems a mistake of 2.1.1 release. let's update it to 2.2.0 directly.